### DISCUSSION TITLE
You can support RSS stylesheets by adding a `index.xsl` file

### DISCUSSION THREAD

**Discussion by kdheepak** [quarto-dev/quarto-cli#4787] [Feature Requests]
By adding a `index.xsl` with appropriate styling, you can make the RSS feed the same style as the webpage. You'll need to add the following to the `index.xml` file.

```
<?xml-stylesheet type="text/xsl" media="screen" href="/index.xsl"?>
```

Here's an example of a `index.xsl` file:

https://github.com/kdheepak/blog/blob/899773dfda874bea7acb1d53e6604079069d9ac4/_quarto/index.xsl

This is a screenshot of my `index.html`, `index.xml` and source of `index.xml` as an example:

<img width="1512" alt="Screenshot 2023-03-13 at 23 02 04" src="https://user-images.githubusercontent.com/1813121/224882323-3f694ff8-73fc-4a35-bd7b-ae2350713984.png">
<img width="1512" alt="Screenshot 2023-03-13 at 23 02 39" src="https://user-images.githubusercontent.com/1813121/224882326-4ab7737e-5d8c-49f9-a65a-28b0bd6a5b81.png">
<img width="1512" alt="Screenshot 2023-03-13 at 23 02 55" src="https://user-images.githubusercontent.com/1813121/224882319-15186447-9479-422b-b2f8-d52b37dcc98a.png">

It'd be ideal if this was autogenerated.

**Comment by dragonstyle** [comment]
This is a nice suggestion and would make a nice feed option. I'll open an issue to track the enhancement and try to get it in for Quarto 1.4!

**Comment by mcanouil** [comment]
Closing in favour of https://github.com/quarto-dev/quarto-cli/issues/4800
